tests/unit/models/test_projects.py

```diff
@@ -40,6 +40,8 @@
 
 # required project data for core24 snaps
 CORE24_DATA = {"base": "core24", "grade": "devel"}
+VALID_DURATIONS = ["10ns", "10us", "10ms", "10s", "10m", "10m4s3us"]
+INVALID_DURATIONS = ["10", "10 s", "10 seconds", "1:00", "invalid"]
```

Comment:
Surprisingly, just an integer isn't a valid duration. The current snapcraft will error out during the packing step:

```
$ snapcraft_edge pack
Generated snap metadata
Cannot pack snap: error: cannot parse snap.yaml: time: missing unit in duration "10"
Detailed information: Command '['snap', 'pack', '--check-skeleton', PosixPath('/root/prime')]' returned non-zero exit status 1.
Failed to execute pack in instance.
Recommended resolution: Run the same command again with --debug to shell into the environment if you wish to introspect this failure.
Full execution log: '/home/lengau/.local/state/snapcraft/log/snapcraft-20250121-181707.415584.log'
```